### PR TITLE
Adds missing pages into sidebar. 

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes https://github.com/ClickHouse/clickhouse-docs/issues/1229

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -286,6 +286,14 @@ const config = {
 						from: '/en/integrations/data-ingestion',
 						to: '/en/integrations',
 					},
+					{
+						from: '/en/integrations/data-ingestion/kafka/code',
+						to: '/en/integrations',
```
Comment (mshustov) [inline docusaurus.config.js:291]: `en/integrations/kafka`?

File: docusaurus.config.js [hunk 2]
```diff
@@ -286,6 +286,14 @@ const config = {
...
+					},
+					{
+						from: '/en/integrations/data-ingestion/kafka/code/connectors',
+						to: '/en/integrations',
```
Comment (mshustov) [inline docusaurus.config.js:295]: `en/integrations/kafka`?

**Review by mshustov — APPROVED** [review]

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit 88e789005dcf42b2f2e55e58db0f2a12af1fd926 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/1739/88e789005dcf42b2f2e55e58db0f2a12af1fd926/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://vercel.com/clickhouse/clickhouse-docs/6mQY5RVPdugqCXdybwpuapNAyseX">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs Check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/1739/88e789005dcf42b2f2e55e58db0f2a12af1fd926/docs_check.html">❌ failure</a></td></tr>
<tbody>
</table>

File: docusaurus.config.js (hunk 2 above)
Comment (johnnymatthews) [inline docusaurus.config.js:295]:
```suggestion
						to: '/en/integrations/kafka',
```

**Pull request merged**